.github/workflows/build-images-base.yaml

```diff
@@ -63,7 +67,7 @@ on:
         type: string
 
 env:
-  IMG_TAGS: ${{ github.sha }} ${{ inputs.kuadrantOperatorTag }}
```

Comment:
so no more sha tags in our images by default... I reckon we are not using them for anything else than the QE builds, no?